stacks/knative/README.md

```diff
@@ -418,6 +417,29 @@ kubectl apply -f <YOUR_KNATIVE_EVENTING_MANIFEST_FILE>
 
 See [Upgrade via the Knative Operator Guide](https://knative.dev/docs/install/upgrade/upgrade-installation-with-operator) for more information on how to upgrade the components, as well as the allowed migration paths.
 
+**Notes:**
+Please also note that if you attempt to upgrade from version `0.x.x` to version `1.x.x` you will get a message in the `knative operator` informing you that: `Version migration is not eligible with message: not supported to upgrade or downgrade across the MAJOR version.` To upgrade to a `MAJOR` version you should remove the existing `Knative Serving` or `Knative Eventing` resources and add new ones as follows:
+
+First remove the Knative Serving resource:
+
+  ```console
+  kubectl delete KnativeServing knative-serving -n knative-serving
+  ```
+
+  or Knative Eventing resource:
+
+  ```console
+  kubectl delete KnativeEventing knative-eventing -n knative-eventing
+  ```
+
+Next, add a new major version in the `spec.version` field of the Eventing/Serving CRDs shown above and apply with `kubectl`:
+
+```console
+kubectl apply -f <YOUR_KNATIVE_SERVING_MANIFEST_FILE>
+
+kubectl apply -f <YOUR_KNATIVE_EVENTING_MANIFEST_FILE>
+```
+
 ## Uninstalling Knative
```

Comment:
The old version (**1.2.2**) is still present in the **Remove the Knative Operator** section. Should be:

```console
OPERATOR_VERSION="1.5.1"
kubectl delete -f "https://github.com/knative/operator/releases/download/knative-v${OPERATOR_VERSION}/operator.yaml"
```

Reply:
Latest commit has everything.